docs/send-data/opentelemetry-collector/data-source-and-configurations/additional-configurations-reference.md

```diff
@@ -0,0 +1,282 @@
+
+As can be observed, while attributes can be present at both **Resource** and **Record level** currently, they are not created equal and should be interpreted separately. The Resource-Level attribute context is much broader, and they identify where data comes from, whereas Record-Level attributes concern just one record, often with many keys and values.
+
+At Sumo Logic, there is a concept of [Fields](https://help.sumologic.com/docs/manage/fields) for log data. Fields offer
+a powerful capability to associate indexable metadata with logs, though only limited number of them can be used at a given time. Also, they need to be defined first.
+
+Looking from the OpenTelemetry standpoint, [Fields](https://help.sumologic.com/docs/manage/fields) are a good match for the **Resource-level** attributes, while Log Record-level attributes are good fit for the [structured representation of the log via JSON](https://help.sumologic.com/docs/search/get-started-with-search/search-basics/view-search-results-json-logs), which
+is automatically supported by Sumo Logic Search.
+
+All **Resource-level** attributes are stored as fields. and any attributes that don't match a defined field will be skipped. You can check the list of ignored fields using the [dropped fields view](https://help.sumologic.com/docs/manage/fields/#view-dropped-fields). When a log contains attributes at the **Record-level** , they are stored as JSON, and if there is a body, it will be stored under the `log` key.
+
+#### Examples
+
+##### Log with both Resource-level and Record-level attributes
+
+Consider the following input log:
+
+```text
+  Resource:
+    Attributes:
+      "indexed-field": "some value"
+  Log:
+    Body: "sample body"
+    Attributes:
+      "log-level-attribute": 42
+```
+
+Such log will be stored as the following set of data at Sumo Logic:
+
+```text
+  Fields:
+    "indexed-field": "some value"
+
+  _raw (JSON): {
+    "log": "sample body",
+    "log-level-attribute": "42"
+  }
+```
+
+ToDo: add screenshots
+
+##### Log with Resource-level attributes only
+
+If no log-level attributes are present, the log body is stored inline. For example, for the following input:
+
+```text
+  Resource:
+    Attributes:
+      "indexed-field": "some value"
+  Log:
+    Body: "sample body"
+```
+
+The output is stored as:
+
+```text
+  Fields:
+    "indexed-field": "some value"
+
+  _raw: "sample body"
+```
+
+ToDo: add screenshots
```

Comment:
commenting out the TODO instances

```suggestion
<!--  ToDo: add screenshots  -->
```

Reply:
Keep them as is because we will fill in the information when available.

Reply:
As long as you can confirm the doc doesn't go out live like this.

Reply:
There is missing information in almost all of the docs, and we are keeping TODO in the docs so that we can easily identify and verify which sections require information.

Reply:
Yikes, I'll speak with Stacy and the OTEL Team